db/models.py

```diff
@@ -1,4 +1,12 @@
+from enum import unique
+
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.exception import ValidationError
```

Comment:
There is a typo in the import statement. It should be `from django.core.exceptions import ValidationError` instead of `from django.core.exception import ValidationError`.